provider/core/src/hello_world.rs

```diff
@@ -157,33 +167,49 @@ make_exportable_provider!(HelloWorldProvider, [HelloWorldV1Marker,]);
 ///     .take_payload()
 ///     .expect("Data should be present");
 ///
-/// assert_eq!(b"{\"message\":\"Hallo Welt\"}", german_hello_world.get());
-pub struct HelloWorldJsonProvider(HelloWorldProvider);
+/// assert_eq!(german_hello_world.get(), br#"{"message":"Hallo Welt"}"#);
+pub struct HelloWorldJsonProvider;
 
+#[cfg(feature = "deserialize_json")]
 impl BufferProvider for HelloWorldJsonProvider {
```

Comment:
The change to use serde_json here is _probably_ fine although it was nifty that previously you could test for a BufferProvider that didn't have all the right features enabled. I guess the benefits outweigh the drawbacks.